lib/include/openamp/version.h

```diff
@@ -21,50 +21,62 @@ extern "C" {
  */
 
 /**
- *  @brief	Library major version number.
+ * @internal
```

Comment:
I missed this one in my list i provided to you...
version.h is not internal, please remove `@internal` tag